Forms/Tools/SeedCalculator.cpp

```diff
@@ -113,7 +123,20 @@ void SeedCalculator::search35()
     ui->progressBar->setRange(0, searcher->getMaxProgress());
 
     auto *timer = new QTimer();
-    connect(timer, &QTimer::timeout, [=] { ui->progressBar->setValue(searcher->getProgress()); });
+    auto startTime = time(0);
+    connect(timer, &QTimer::timeout, [=] {
```

Comment:
"Any slower", yes, since something has to be calculated. But it is realy nothing compared to the calculation of the seeds.

Lets say that around 1000 seeds are checked every second (at least on my PC). Checking a single seed requires running the searchSeed method which has around 250 lines of code. That means that 250,000 lines of code are executed every second. The timer that updates the exstimated time is only activated every second. So there are additionally 10 lines of code every second.

That means that for a calculation that used to take 1 hour there are now 10 * 60 * 60 = 36,000 additional lines of code. Given that we execute 250,000 lines each second these extra calculations wouldn't even take an extra second.

So I would put up with that to know how long the calculation is going to take.